App/Client.xcodeproj/project.pbxproj

```diff
@@ -31,7 +31,6 @@
 		27FA217F2837FB0100FA2C45 /* BraveShared in Frameworks */ = {isa = PBXBuildFile; productRef = 27FA217E2837FB0100FA2C45 /* BraveShared */; };
 		27FA21812837FB0100FA2C45 /* BraveWallet in Frameworks */ = {isa = PBXBuildFile; productRef = 27FA21802837FB0100FA2C45 /* BraveWallet */; };
 		27FA21832837FB0100FA2C45 /* Data in Frameworks */ = {isa = PBXBuildFile; productRef = 27FA21822837FB0100FA2C45 /* Data */; };
-		27FA21852837FB0100FA2C45 /* GuardianVPN in Frameworks */ = {isa = PBXBuildFile; productRef = 27FA21842837FB0100FA2C45 /* GuardianVPN */; };
```

Comment:
Just remembered this: Make sure you update the `l10n` project so that `GRDServerManager.m` has the right path otherwise its string won't be exported and localized

Reply:
We do not use any strings from those files, I will double check

Reply:
oops, i will see if its being used and remove if not needed

Reply:
I removed that localized string